base/acme/webapps/acme/index.jsp

```diff
@@ -22,88 +30,57 @@ $(function() {
 
   <header class="pf-c-page__header">
     <div class="pf-c-page__header-brand">
+      <div class="pf-c-page__header-brand-toggle">
+        <button id="sidebar-toggle" class="pf-c-button pf-m-plain" type="button" aria-label="Global navigation" aria-controls="sidebar">
+          <i class="fas fa-bars" aria-hidden="true"></i>
+        </button>
+      </div>
       <a class="pf-c-page__header-brand-link">ACME Responder</a>
     </div>
+    <div class="pf-c-page__header-tools">
+      <div class="pf-c-dropdown" style="--pf-global--BorderWidth--sm: 0px;">
+        <button id="profile-menu" class="pf-c-dropdown__toggle" type="button" aria-expanded="false">
+          <span class="pf-c-dropdown__toggle-image">
+            <img class="pf-c-avatar" alt="Avatar image" src="css/assets/images/img_avatar.svg">
+          </span>
+          <span id="profile-fullName" class="pf-c-dropdown__toggle-text"></span>
+          <span class="pf-c-dropdown__toggle-icon">
+            <i class="fas fa-caret-down" aria-hidden="true"></i>
+          </span>
+        </button>
+        <ul aria-labelledby="profile-menu" class="pf-c-dropdown__menu pf-m-align-right" hidden="hidden">
+          <li id="login-menu" hidden="hidden">
+            <a class="pf-c-dropdown__menu-item" href="#">Log In</a>
+          </li>
+          <li id="logout-menu" hidden="hidden">
+            <a class="pf-c-dropdown__menu-item" href="#">Log Out</a>
+          </li>
+        </ul>
+      </div>
+    </div>
   </header>
 
+  <div class="pf-c-page__sidebar">
+    <div class="pf-c-page__sidebar-body">
+      <nav id="sidebar" class="pf-c-nav" aria-label="Global">
+        <ul class="pf-c-nav__list">
+          <li id="home-menu" class="pf-c-nav__item">
+            <a href="#" class="pf-c-nav__link pf-m-current" aria-current="page">Home</a>
+          </li>
```

Comment:
Could we add `login`/`logout` here as well? I think that'd be a good usability improvement.

Reply:
OK, I added login/logout menus as well on the sidebar. I renamed the Help page into Services and moved it above the Configuration page so the menu doesn't change much when you're logged in or out.